Trip Planner/cli/trips_pendingApproval.php

```diff
@@ -25,7 +25,8 @@
 use Gibbon\Module\TripPlanner\Domain\TripGateway;
 use Gibbon\Module\TripPlanner\Domain\ApproverGateway;
 
-require getcwd().'/../gibbon.php';
+//require getcwd().'/../gibbon.php';
```

Comment:
No need to leave old code commented out, we can always use git history if we need to see what it was before.